УсловныеОператоры2/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace УсловныеОператоры2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите символ");
+
+            string str = Console.ReadLine();
+            for (int i = 0; i < str.Length; i++)
+            {
+
+                if (char.IsUpper(str[i]))
+                    Console.WriteLine("{0} - заглавная буква.", str[i]);
+
+                else if (char.IsLower(str[i]))
+                    Console.WriteLine("{0} - строчная буква.", str[i]);
+
+                else if (char.IsDigit(str[i]))
+                    Console.WriteLine("{0} - число.", str[i]);
```

Comment:
Некорректное сообщение.
```suggestion
                    Console.WriteLine("{0} - цифра.", str[i]);
```